### PR TITLE
Update to Laravel 8

### PR DESCRIPTION
**Pull request by eldor**
## Proposed Changes

  - Update to Laravel 8

### CODE REVIEW

**Comment by eldor** [comment]
Your github actions needed to be change to correct PHP version for Laravel 8 and also updated the PHPUnit and testbench

**Comment by eldor** [comment]
@elkebirmed any change this could be looked at so this good package can be used with Laravel 8?

**Comment by elkebirmed** [comment]
Supporting Laravel 8 is good but dropping support of lower versions is very bad.
Can you add Laravel 8 to the chain of versions?

**Comment by eldor** [comment]
@elkebirmed good idea, its updated now

File: composer.json [hunk 1]
```diff
@@ -17,13 +17,13 @@
         "Localizater"
     ],
     "require": {
-        "php": "^7.2",
-        "illuminate/support": "~5|~6|~7"
+        "php": "^7.3",
```
Comment (elkebirmed) [inline composer.json:20]: ^7.2|^7.3

**Review by elkebirmed — COMMENTED** [review]
Add PHP 7.2

**Comment by elkebirmed** [comment]
It's seems like Laravel 8 requires PHP 7.3 and above !!!

**Pull request merged**